### PR TITLE
chore: remove nango from connectors

### PR DESCRIPTION
**Pull request by fontanierh**
## Description

Pretty self explanatory

## Risk

Not too risky

### CODE REVIEW

File: connectors/package.json [hunk 1]
```diff
@@ -17,7 +17,6 @@
     "@dust-tt/types": "file:../types",
     "@microsoft/microsoft-graph-client": "^3.0.7",
     "@microsoft/microsoft-graph-types": "^2.40.0",
-    "@nangohq/node": "^0.21.4",
```
Comment (flvndvd) [inline connectors/package.json:20]: 😘 🫡

File: connectors/src/connectors/confluence/temporal/activities.ts [hunk 2]
```diff
@@ -63,24 +58,13 @@ async function fetchConfluenceConnector(connectorId: ModelId) {
 }
 
 async function getConfluenceAccessTokenWithThrow(connectionId: string) {
```
Comment (flvndvd) [inline connectors/src/connectors/confluence/temporal/activities.ts:60]: I was gonna remove this function that was need while we were phasing out Nango. Now, we can just call `getOAuthConnectionAccessTokenWithThrow` where it's needed.

File: connectors/src/connectors/google_drive/lib/config.ts [hunk 3]
```diff
@@ -1,9 +1,6 @@
 import { EnvironmentConfig } from "@dust-tt/types";
 
 export const googleDriveConfig = {
-  getRequiredNangoGoogleDriveConnectorId: (): string => {
-    return EnvironmentConfig.getEnvVariable("NANGO_GOOGLE_DRIVE_CONNECTOR_ID");
-  },
   getRequiredGoogleDriveClientId: (): string => {
```
Comment (spolu) [inline connectors/src/connectors/google_drive/lib/config.ts]: I think we can remove it all?

File: connectors/src/lib/temporal_monitoring.ts [hunk 4]
```diff
@@ -114,25 +113,6 @@ export class ActivityInboundLogInterceptor
     } catch (err: unknown) {
       error = err;
 
-      if (
```
Comment (spolu) [inline connectors/src/lib/temporal_monitoring.ts:117]: These will likely come back at us but we'll see :)

**Review by spolu — APPROVED** [review]
LGTM

**Comment by spolu** [comment]
Nice!

**Review by flvndvd — APPROVED** [review]
LGTM, left 1 small comment

File: connectors/src/connectors/confluence/temporal/activities.ts (hunk 2 above)
Comment (fontanierh) [reply connectors/src/connectors/confluence/temporal/activities.ts:60]: Yeah we can def have one single function for every connector.

Will do in separate PR (unless you're already doing? )

**Pull request merged**